.github/workflows/build.yml

```diff
@@ -38,10 +35,25 @@ jobs:
         with:
           java-version: "adopt@1.8"
 
-      - name: Build and run tests
-        run: sbt "project agent" ++${{matrix.scala}} test doc "project model" ++${{matrix.scala}} test doc
+      - name: Build and run unit tests
+        run: sbt "project agent" test doc "project model" test doc
 
-  build-server:
+  build-server-unit:
```

Comment:
`-unit` is not present in the `build-agent-and-model` job, perhaps consider making it consistent